iina/ControlBarView.swift

```diff
@@ -15,7 +15,8 @@ class ControlBarView: NSVisualEffectView {
   var isDragging: Bool = false
 
   override func awakeFromNib() {
-    self.layer?.cornerRadius = 6
+    self.layer?.cornerRadius = 4
```

Comment:
I personally think 6pt is better, and actually IINA prefers larger corner radius not only for OSC during early development.